### DISCUSSION TITLE
Welcome to capa Discussions!

### DISCUSSION THREAD

**Discussion by williballenthin** [mandiant/capa#382] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

(this is willi:) I've clicked through the default Discussions config to create this. Let's see what we can use this Github feature for.

**Comment by williballenthin** [comment]
I think we could convert a bunch of the discussion issues to real "discussions":

![image](https://user-images.githubusercontent.com/156560/101530133-c2997000-394e-11eb-9ec6-e87d27fb4612.png)

https://github.com/fireeye/capa/issues?q=is%3Aissue+is%3Aopen+discussion

**Comment by musaabimran** [comment]
I like the capa tool for static analysis
